src/components/RadioButton.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { CheckBox as CheckBoxNative, CheckBoxProps, Text } from 'react-native-elements';
+
+import { palette, typography } from 'app/styles';
+
+interface Props<T> {
+  title: string;
+  subtitle: string;
+  value: T;
+  selectedValue: T;
+  onPress?: (value: T) => void;
+  testID?: string;
+}
+
+export const RadioButton = <T extends string>(props: Props<T>) => {
+  const [value] = useState(props.value);
+
+  const onPressHandler = () => {
+    props.onPress && props.onPress(value);
+  };
+
+  const isValueSelected = (value: T, selectedValue: T) => value === selectedValue;
+
+  const renderTitle = (title: string, subtitle: string) => (
+    <View style={styles.radioButtonContent}>
+      <Text style={styles.radioButtonTitle}>{title}</Text>
+      <Text style={styles.radioButtonSubtitle}>{subtitle}</Text>
+    </View>
+  );
+
+  return (
+    <CheckBoxNative
+      title={renderTitle(props.title, props.subtitle)}
+      checked={isValueSelected(value, props.selectedValue)}
+      containerStyle={styles.containerStyle}
+      wrapperStyle={styles.wrapperStyle}
+      iconType="material"
```

Comment:
We don't have RadioButton icon image in assets. Not sure how should I handle this situation

Reply:
you can ask UX to generate you the icon for it

Reply:
Done